libp2p/protocols/secure/secio.nim

```diff
@@ -334,7 +335,7 @@ method handshake*(s: Secio, conn: Connection, initiator: bool = false): Future[S
                        remoteNonce)
   trace "Remote proposal", schemes = remoteExchanges, ciphers = remoteCiphers,
                            hashes = remoteHashes,
-                           pubkey = toHex(remoteBytesPubkey), order = order,
+                           pubkey = remoteBytesPubkey.shortHexDump, order = order,
```

Comment:
`shortHexDump` here is not useful here because in such case you will lose public key type information.